aws/cloudfront_distribution_configuration_structure.go

```diff
@@ -51,6 +50,11 @@ func expandDistributionConfig(d *schema.ResourceData) *cloudfront.DistributionCo
 		Origins:              expandOrigins(d.Get("origin").(*schema.Set)),
 		PriceClass:           aws.String(d.Get("price_class").(string)),
 	}
+	if v, ok := d.GetOk("ordered_cache_behavior"); ok {
+		distributionConfig.CacheBehaviors = expandCacheBehaviors(v.([]interface{}))
+	} else {
+		distributionConfig.CacheBehaviors = expandCacheBehaviorsDeprecated(d.Get("cache_behavior").(*schema.Set))
```

Comment:
Old behavior is kept under a expandCacheBehaviorsDeprecated function, which contains the exact same previous code.